adding mount mode for volumes

srgrn opened the pull request:
I had a problem since i mounted some volumes as readonly in my compose files 
so the mount line was 
- src:dst:ro

this commit actually makes all volumes to be mounted as readwrite in a verbose way by adding readonly=0 for all volumes and readonly=1 for the ro volumes

Commit:
adding mount mode for volumes

ddrozdov commented on docker_compose_swarm_mode.py:
```diff
@@ -141,15 +141,19 @@ def networks():
+                        splitted_volume = volume.split(':')
+                        src = splitted_volume.pop(0)
+                        dst = splitted_volume.pop(0)
+                        mode = 0
```
Let's call this variable `readonly`.

ddrozdov commented on docker_compose_swarm_mode.py:
```diff
@@ -141,15 +141,19 @@ def networks():
+                        src = splitted_volume.pop(0)
+                        dst = splitted_volume.pop(0)
+                        mode = 0
+                        if a and a[0] == 'ro':
```
What is `a`? Should it be `splitted_volume` instead?

srgrn commented:
Sorry I reverted a change after checking. 
Ill fix it in a few minutes 

Sent from BlueMail

On Oct 6, 2016, 13:30, at 13:30, Dmitry notifications@github.com wrote:

> ddrozdov requested changes on this pull request.
> 
> > @@ -141,15 +141,19 @@ def networks():
> 
> ```
>             def volumes():
>                 for volume in value:
> ```
> 
> ## \-                        src, dst = volume.split(':')
> -                        splitted_volume = volume.split(':')
> -                        src = splitted_volume.pop(0)
> -                        dst = splitted_volume.pop(0)
> -                        mode = 0
> 
> Let's call this variable `readonly`.
> 
> > @@ -141,15 +141,19 @@ def networks():
> 
> ```
>             def volumes():
>                 for volume in value:
> ```
> 
> ## \-                        src, dst = volume.split(':')
> -                        splitted_volume = volume.split(':')
> -                        src = splitted_volume.pop(0)
> -                        dst = splitted_volume.pop(0)
> -                        mode = 0
> -                        if a and a[0] == 'ro':
> 
> What is `a`? Should it be `splitted_volume` instead?
> 
> ## 
> 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/ddrozdov/docker-compose-swarm-mode/pull/19#pullrequestreview-3084066

Commit:
fixing my errors and including arrayed commands

srgrn commented:
I have fixed the bugs and also added support for CMD in array format so i want to open a new pull request
